tools/data-handler/src/move.ts

```diff
@@ -36,12 +39,18 @@ export class Move {
   /**
    * Moves card from 'destination' to 'source'.
    * @param source source card to move
-   * @param destination destination card where source card will be moved to; or 'root'
+   * @param destination destination card where source card will be moved to; or to root
    */
   public async moveCard(source: string, destination: string) {
+    if (source === ROOT) {
+      throw new Error('Cannot move "root"');
+    }
+    if (source === destination) {
```

Comment:
Shouldn't we also check, that destination is not under the source card?

Reply:
> Shouldn't we also check, that destination is not under the source card?
Yes, I'll add that as a check as well.

Reply:
Fixed